.github/workflows/main.yml

```diff
@@ -77,8 +77,8 @@ jobs:
       # Build only the core to be able to run the static code analysis
       - name: Compile XmippCore
         run: |
-          cp ${BUILD_DIR}/xmipp.conf ./install
-          build-wrapper-linux-x86-64 --out-dir bw-outputs scons . -j4
+          cp ${BUILD_DIR}/xmipp.conf ./
+          build-wrapper-linux-x86-64 --out-dir bw-outputs /usr/bin/env python3 $(which scons) . -j4
```

Comment:
works with python 2.7 out of scipion3 env?

Reply:
No. Python3 is required (also for scipion, so it should be there)

Reply:
Also notice that this change is only for the github action script

Reply:
ok, thanks